bridge/qb/server/events.lua

```diff
@@ -38,4 +38,13 @@ RegisterServerEvent('baseevents:leftVehicle', function(veh, seat, modelName, net
         event = 'Left'
     }
     TriggerClientEvent('QBCore:Client:VehicleInfo', src, data)
+end)
+
+-- Player
+RegisterServerEvent('qbx_core:server:onSetMetaData', function (meta, _, val, source)
+    if meta == 'thirst' or meta == 'hunger' then return end
+    local playerState = Player(source).state
+    if val == playerState[meta] then return end
+    if not playerState.isLoggedIn then return end
+    playerState:set(meta, val, true)
```

Comment:
Why is this in bridge. Shouldn't this be within the main core files?

Reply:
Because it's Backwards compatibility with Scripts that still use the setMetadata way? (ours still do but it's temporary)

Reply:
The bridge is for backwards compatibility with qb. This is backwards compatibility with previous versions of Qbox.

Reply:
> The bridge is for backwards compatibility with qb. This is backwards compatibility with previous versions of Qbox.

It's backwards compat with qb, and qbox.

Reply:
Shouldn't someone be able to turn off the bridge using config if they aren't getting the core object? The idea of the bridge is to provide backwards compatibility to qb, not previous versions of Qbox.

Reply:
Hmm ok i see, i'll move it to regular server/events